lib/shoulda/matchers/active_model/validate_absence_of_matcher.rb

```diff
@@ -143,12 +143,6 @@ def reflection
             @subject.class.reflect_on_association(@attribute)
         end
 
-        def array_column?
```

Comment:
Since this PR is focused on updating the `validate_length_of` matcher, should we do this in a different PR? This will just override the method you've added to ValidationMatcher, but that's okay for now.

Reply:
I decided to do it in this PR to avoid repetition of `array_column?` and to avoid forgetting to remove it later 😅, I'll roll it back and will do the change on another PR after this one gets merged 👍